lib/utils.sh

```diff
@@ -0,0 +1,22 @@
+function print_error() {
+   echo "$1"
+   echo ""
+   echo "$CMD_USAGE"
+   exit 1
+}
+
+function override_aws_credentials() {
+    AWS_VARS="AWS_PROFILE AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS AWS_SECRET_ACCESS AWS_REGION"
```

Comment:
What is `AWS_SECRET_ACCESS` ? Env vars used by awscli are found here: https://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html